AcadeMeData/tests/test_views.py

```diff
@@ -1,15 +1,25 @@
-from django.template.loader import render_to_string
+from pytest_django.asserts import assertTemplateUsed
 import pytest
-from AcadeMe import views
+from AcadeMeData.models import University, Degree
 
 
 @pytest.mark.django_db
-def test_homepage_html(client):
-    response = views.homePage(client)
-    assert response.content.decode() == render_to_string('landing/homepage.html')
+def test_homepage_html(client, user_example):
+    all_universities = University.objects.all()
+    all_degrees = Degree.objects.all()

```

Comment:
Consider adding fixture for here, that do the same in conftest

Reply:
Done